content/pages/community.html

```diff
@@ -0,0 +1,125 @@
+                    situation, please refer to the Commercial Support section
+                    below.</em>
+                </p>
+
+                <h1>Contributing</h1>
+                <p>
+                    We welcome community contributions! Please see the
+                    <code>CONTRIBUTING.md</code> file in the respective Kiwi TCMS repository
+                    for instructions on how to submit changes. If you are planning on making
+                    more elaborate or controversial changes, please discuss them on the mailing
+                    list before sending a pull request.
+                </p>
+                <p>
+                    <strong>Development mailing list:</strong>
+                    <a href="https://groups.google.com/forum/#!forum/prometheus-developers">kiwitcms-developers</a> – for discussions around Kiwi TCMS development
+                </p>
+
+                <h1>Project Governance</h1>
+                <p>
+                    Kiwi TCMS is an independent open-source project and not controlled by
+                    any single company. To emphasize this we joined the
+                    <a href="https://cncf.io/">Cloud Native Computing Foundation</a> in 2016.
+                </p>
+                <p>
+                    Within the project, we make decisions based on <a href="/governance/">these rules</a>.
+                </p>
+
+                <h1>Code of Conduct</h1>
+                <p>
+                    To make Kiwi TCMS a welcoming and harassment-free experience for everyone, we follow the
+                    <a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">CNCF Code of Conduct</a>.
+                </p>
+
+                <h1>Commercial Support</h1>
+                <p>
+                    This is a list of third-party companies and individuals who provide products 
+                    or services related to Kiwi TCMS. Kiwi TCMS is an independent open source 
+                    project which does not endorse any company.
+                </p>
+                <ul>
+                    <li><a href="https://www.robustperception.io">Robust Perception</a></li>
+                    <li><a href="https://juliusv.com">Julius Volz</a> (independent contractor)</li>
+                    <li><a href="https://container-solutions.com">Container Solutions</a></li>
+                    <li><a href="https://tincho.org">Martín Ferrari</a> (independent contractor)</li>
+                    <li><a href="https://kausal.co">Kausal</a></li>
+                    <li><a href="https://fabxc.org">Fabian Reinartz</a> (independent contractor)</li>
+                </ul>
+
+                <h1>Acknowledgements</h1>
+                <p>
+                    Kiwi TCMS was initially started by
+                    <a href="https://www.redhat.com/en">Red Hat, Inc.</a>. The later development is being done by <a href="http://mrsenko.com/">Mr. Senko, Ltd.</a>.
```

Comment:
Update the text a bit to be more clear about the history of the project, keep the links:
```
Kiwi TCMS was initially started by Red Hat, Inc. under the name Nitrate. It was intended as a replacement for Mozilla's Testopia and has grown on its own since then. The latest development is being done by the Kiwi TCMS team with sponsorship by Mr. Senko, Ltd.
```